samples/snippets/src/main/java/com/example/bigquery/AuthorizeDataset.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Samples require integration tests to ensure they work. See: https://github.com/googleapis/java-bigquery/tree/main/samples/snippets/src/test/java/com/example/bigquery

Reply:
sure, added AuthorizeDatasetIT